src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -172,9 +172,23 @@ private void initCommonVariables(InstructorAttributes instructor, String selecte
             questionToDetailsMap.put(question, questionDetails);
         }
         
+        this.sections = sectionNames();
+        
         displayableFsName = sanitizeForHtml(bundle.feedbackSession.feedbackSessionName);
         displayableCourseId = sanitizeForHtml(bundle.feedbackSession.courseId);
     }
+
+    private List<String> sectionNames() {
```

Comment:
I didn't measure it, but it's not going to be significant with regards to the page load time since most of the time is due to processing feedback responses, but this extra query is done multiple times on a single access to the results page, on the first load and for every question in the question view. This will probably save a bit of $ over time for larger classes.